screens/EpisodeDisplayScreen.tsx

```diff
@@ -0,0 +1,142 @@
+import { useNavigation } from "@react-navigation/native";
+import React from "react";
+import { StyleSheet, ScrollView } from "react-native";
+import { Text, View } from "../components/Themed";
+import { Episode } from "../types";
+
+interface EpisodeProps {
+  episode: Episode;
+  index: number;
+}
+
+function EpisodeDisplay({ episode, index }: EpisodeProps) {
+  const convertMinutesToTime = (minutes: number) => {
+    const date = new Date(minutes * 60 * 1000);
+    return date.toLocaleTimeString("en-US", {
+      hour: "numeric",
+      minute: "2-digit",
+    });
+  };
+
+  return (
+    <View key={`${episode.name}${index}`} style={episodeStyles.container}>
+      <View style={episodeStyles.numberContainer}>
+        <Text style={episodeStyles.numberText}>{index}</Text>
+      </View>
+      <View style={episodeStyles.infoContainer}>
+        <Text style={episodeStyles.episodeTitleText} numberOfLines={1}>
+          {episode.name}
+        </Text>
+        <Text style={episodeStyles.episodeInfoText}>{`${convertMinutesToTime(
+          episode.startTime
+        )} - ${convertMinutesToTime(episode.endTime)}`}</Text>
+        <Text style={episodeStyles.episodeInfoText} numberOfLines={1}>{`with ${
+          episode.initials.trim() || `N/A`
+        }`}</Text>
```

Comment:
Can we put as much of this text not in javascript template as possible? So the second would be like 'with {episode.initials.trim() || 'N/A'}'